tests/test_diffeq/test_perturbedsolvers/test_perturbation_functions.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+from probnum.diffeq.perturbedsolvers import _perturbation_functions
+
+random_state = np.random.mtrand.RandomState(seed=1)
+
+
+def test_perturb_uniform():
+    random_state = np.random.mtrand.RandomState(seed=1)
+    step = 0.2
+    noise_scale = 1
+    order = 4
```

Comment:
the parameters "step, noise_scale, order" would make good fixtures (https://docs.pytest.org/en/stable/fixture.html). If you dont know what I mean, let me know offline :)